lib/bolt/config.rb

```diff
@@ -110,7 +110,7 @@ def normalize_interpreters(interpreters)
     def normalize_log(target)
       return target if target == 'console'
       target = target[5..-1] if target.start_with?('file:')
-      'file:' + File.expand_path(target)
+      'file:' + File.expand_path(target, @boltdir.path)
```

Comment:
We should probably get this on @future? Is that possible

Reply:
Updated - also  I think you accidentally pinged a random github user named 'future' :D